src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -64,13 +65,13 @@
     private static Date earliestSinceDate = null;
 
     /**
-     * Generates the authorship and commits JSON file for each repo in {@code configs} at {@code outputPath}, as
-     * well as the summary JSON file of all the repos.
+     * Generates and returns a list of paths to the authorship and commits JSON file for each repo in {@code configs}
+     * at {@code outputPath}, as well as the summary JSON file of all the repos.
```

Comment:
A forcefully condensed message adds more confusion and is not advised.
Retain the original message and add another line `@return the list of filepaths that were generated` instead.